alg-excer3.js

```diff
@@ -0,0 +1,33 @@
+/*Create a function that takes a string and returns a new string with all vowels removed.
+Examples
+removeVowels("I have never seen a thin person drinking Diet Coke.")
+➞ " hv nvr sn  thn prsn drnkng Dt Ck."*/
+
+const removingVowel = str => {
+	var newString = "";
```

Comment:
nit tabbing otherwise looks good!

Reply:
not showing in my VSC, already confirmed